clusters/app.ci/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -0,0 +1,16 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: dedicated-admins
```

Comment:
NIP: Could we use the same group name as api.ci and build01?

```bash
oc get group ci-admins
NAME        USERS
ci-admins   ...

oc --context build01 get group ci-admins
NAME        USERS
ci-admins   ...
```

/lgtm

/hold

in case you want to rename the group

Reply:
I imagine this gets tied into the OSD mechanisms so it has to be named as such

/hold cancel

Reply:
Unfortunatly not, because there is a set of RoleBindings being created that are bound to this group.
.hold

Reply:
/hold cancel